api/v1/cluster_webhook.go

```diff
@@ -1195,6 +1195,15 @@ func (r *Cluster) validateConfiguration() field.ErrorList {
 		}
 	}
 
+	if r.Spec.Instances > 1 && r.Spec.PostgresConfiguration.Parameters[postgres.ParameterWalLogHints] == "off" {
+		result = append(
```

Comment:
I think this needs to be revised.

From the Postgres docs about the config file:
> Boolean: Values can be written as on, off, true, false, yes, no, 1, 0 (all case-insensitive) or any unambiguous prefix of one of these.

That means that a user could set it to "0", "no", or even "OFF" and go ahead with a broken configuration.

If we don't want to parse the value, we can use `!= "on"` in the condition, forcing the user to use `on` as the only valid value for true.

Also, is there any possibility that the parameter is not present and we compare an empty value?

Reply:
Same thing with archive_mode, where we choose this behavior.

If we like something different, let's address this but not in this PR.

Reply:
So, we should also change the `archive_mode` check to be more precise. Doing that in another PR is fine for me. Let's merge this feature and create another issue to improve the checks involving boolean parameters.

Reply:
Will you create an issue for that @mnencia?

Reply:
When you do it, assign it to me.